packages/core/src/RenderingEngine/StackViewport.ts

```diff
@@ -459,9 +471,15 @@ class StackViewport extends Viewport {
     }
 
     try {
-      return [createSharpeningRenderPass(this.sharpening)];
+      if (this.sharpening > 0) {
```

Comment:
I dont think it should be else if, are we not able to apply multiple? What if the viewport has both above 0 for whatever reason?